commit 84920e9c5844b3848b19f4ce25e054e55b8607bd (from c80a78bb704604e568a9931f2d6f1c80c2395e35)
Merge: c80a78b 31bb3fb
Author: daid <daid303@gmail.com>
Date:   Fri Apr 12 13:45:15 2013 -0700

    Merge pull request #449 from Stefanowicz/Marlin_v1
    
    Fix: Build error when EEPROM_SETTINGS is defined and PIDTEMP is undefined

diff --git a/Marlin/ConfigurationStore.cpp b/Marlin/ConfigurationStore.cpp
index 574c438..4e11c44 100644
--- a/Marlin/ConfigurationStore.cpp
+++ b/Marlin/ConfigurationStore.cpp
@@ -72,9 +72,11 @@ void Config_StoreSettings()
     EEPROM_WRITE_VAR(i,Ki);
     EEPROM_WRITE_VAR(i,Kd);
   #else
-    EEPROM_WRITE_VAR(i,3000);
-    EEPROM_WRITE_VAR(i,0);
-    EEPROM_WRITE_VAR(i,0);
+		float dummy = 3000.0f;
+    EEPROM_WRITE_VAR(i,dummy);
+		dummy = 0.0f;
+    EEPROM_WRITE_VAR(i,dummy);
+    EEPROM_WRITE_VAR(i,dummy);
   #endif
   char ver2[4]=EEPROM_VERSION;
   i=EEPROM_OFFSET;

